app/components/subheader/model.js

```diff
@@ -4,6 +4,8 @@ const sanitize = require('../../services/universal/sanitize');
 
 module.exports.save = function (ref, data) {
   data.text = sanitize.validateTagContent(sanitize.toSmartText(data.text || ''));
-
+  data.css = data.type;
```

Comment:
Do you need to assign `type` to a new property? It looks like it's edited through a radio button and isn't manipulated in any way